samples/legacy_support/All_ID_Mapping_Original.csv

```diff
@@ -2579,7 +2579,7 @@ id,Name,LegacyKey,MainIcon,Modifier1,Modifier2,ExtraIcon,FullFrame,GeometryType,
 2577,SIGINT.SPC.SIGINC.RAD.MFN,I-P-SRM---,I-P-SRM---,,,,,Point,,,
 2578,SIGINT.SPC.SIGINC.RAD.TGTAQ,I-P-SRT---,I-P-SRT---,,,,,Point,,,
 2579,SIGINT.SPC.SIGINC.RAD.SPC,I-P-SRS---,I-P-SRS---,,,,,Point,,,
-2580,I-P-SRU---.svg,I-P-SRU---,I-P-SRU---,,,,,Point,,,
+2580,SIGINT.SPC.SIGINC.RAD.UNK,I-P-SRU---,I-P-SRU---,,,,,Point,,,
```

Comment:
I didn't realize these were using the hierarchy codes (line above) vs. human-readable names (see line below) - I opened a question about this here: https://github.com/Esri/joint-military-symbology-xml/issues/292